DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/register_map_and_streaming_interfaces/README.md

```diff
@@ -0,0 +1,393 @@
+
+
+# `Register Map and Streaming Interfaces` sample
```

Comment:
To be more aligned with the other samples, I think that this section should only contain a one or two sentences summary of what that sample contains.
Then the "optimized for" table, the prerequisites, and finally the purpose section.
From there you can go in as many details as you want